server/vulnerabilities/nvd/cve.go

```diff
@@ -9,17 +9,25 @@ import (
 	"path/filepath"
 	"regexp"
 	"runtime"
+	"strings"
 	"sync"
 	"time"
 
+	"github.com/Masterminds/semver"
 	"github.com/facebookincubator/nvdtools/cvefeed"
+	feednvd "github.com/facebookincubator/nvdtools/cvefeed/nvd"
+	"github.com/facebookincubator/nvdtools/cvefeed/nvd/schema"
 	"github.com/facebookincubator/nvdtools/providers/nvd"
 	"github.com/facebookincubator/nvdtools/wfn"
+	"github.com/fleetdm/fleet/v4/server/contexts/ctxerr"
 	"github.com/fleetdm/fleet/v4/server/fleet"
 	kitlog "github.com/go-kit/log"
 	"github.com/go-kit/log/level"
 )
 
+// Define a regex pattern for semver (simplified)
+var semverPattern = `^v?(\d+\.\d+\.\d+)`
```

Comment:
What I meant was:
```go
var semverPattern = regexp.MustCompile(semverPattern)
```

Reply:
haha my mistake, getting sloppy late in the day